src/org-unit-tree/OrgUnitTree.component.js

```diff
@@ -42,7 +41,7 @@ class OrgUnitTree extends React.Component {
                 : undefined,
             loading: false,
         };
-        if (props.root.children instanceof ModelCollection) {
+        if (!props.root.children instanceof Model) {
```

Comment:
Did you test if this worked? Since this statement should always return false. As `!props.root.children` will never be an instance of `Model`.

Which basically means that it will never run the `toArray()` when a `ModelCollection` is passed. Perhaps it could be something like:

```js
if (!Array.isArray(props.root.children.toArray) && typeof props.root.children.toArray === 'function') {
```